packages/js/components/changelog/add-35851-tree-control-a11y

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: dev
+
+Add a11y support for the tree-control
```

Comment:
Instead of `tree-control`, consider `Tree` component

Reply:
Thanks @mattsherman. It's done now.